src/joystick/linux/SDL_sysjoystick.c

```diff
@@ -182,6 +193,30 @@ static int GuessIsJoystick(int fd)
     return 0;
 }
 
+static int GuessIsSensor(int fd)
+{
+    unsigned long evbit[NBITS(EV_MAX)] = { 0 };
+    unsigned long keybit[NBITS(KEY_MAX)] = { 0 };
+    unsigned long absbit[NBITS(ABS_MAX)] = { 0 };
+    unsigned long relbit[NBITS(REL_MAX)] = { 0 };
+    int devclass;
+
+    if ((ioctl(fd, EVIOCGBIT(0, sizeof(evbit)), evbit) < 0) ||
+        (ioctl(fd, EVIOCGBIT(EV_KEY, sizeof(keybit)), keybit) < 0) ||
+        (ioctl(fd, EVIOCGBIT(EV_REL, sizeof(relbit)), relbit) < 0) ||
+        (ioctl(fd, EVIOCGBIT(EV_ABS, sizeof(absbit)), absbit) < 0)) {
+        return 0;
+    }
+
+    devclass = SDL_EVDEV_GuessDeviceClass(evbit, absbit, keybit, relbit);
+
+    if (devclass & SDL_UDEV_DEVICE_ACCELEROMETER) {
```

Comment:
We should maybe rename `ACCELEROMETER` to something like `MOTION_CONTROL` since it really means "accelerometer and/or gyro".

Reply:
... or maybe we shouldn't, since it's a 1:1 representation of udev's `ID_INPUT_ACCELEROMETER`.